test-resources/agents-schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Ably Agents Definition",
+    "id": "https://schemas.ably.com/json/asset-common/Agents",
```

Comment:
Should we set up `schemas.ably.com`? We're also going to have JSON schemas for plugins (see [here](https://ably.atlassian.net/wiki/spaces/ENG/pages/1197342737/Ably+Plugins+Architecture#Parameters)), and so far we've referenced the schemas under the fictitious `plugins.ably.com` domain, but would be good to standardise this (separate from this PR, of course).

Reply:
This isn't the first time this has come up. I'm trying to find previous conversation(s)...

Reply:
Ah, yes: https://github.com/ably/ably-asset-tracking-common/blob/170429211965e098e211bc117d29eeceef405e64/test-resources/geo/journey-schema.json#L9

Reply:
Ideally I think we would want to do https://github.com/ably/ably-common/issues/70 first and then have a formal release procedure, which could also publish schemas out via S3 or whatever. (apologies for monologue 😆)

Reply:
I'm happy for us to set up the subdomain and host these at some point but personally I don't see the problem with them being fictitious either. I do see the value in standardising this though, so I would be in favour of changing the plugin schemas to use `schemas.ably.com`.

Reply:
Sounds good, perhaps we'll want a `isSchema` option (or equivalent) for the sdk-upload-action so repositories can publish their own schemas to `schemas.ably.com` using GitHub actions?